src/types/profile-derived.js

```diff
@@ -34,8 +34,8 @@ export type IndexIntoCallNodeTable = number;
  * `docs-developer/call-nodes-in-cpp.md`.
  */
 export type CallNodeTable = {
-  prefix: Int32Array,
-  func: Int32Array,
+  prefix: Int32Array, // IndexIntoCallNodeTable -> IndexIntoCallNodeTable
```

Comment:
Would `prefix: Int32Array, // IndexIntoCallNodeTable -> IndexIntoCallNodeTable | -1` be clearer?

Reply:
clearer, I don't know, but more right for sure. Let me fire up a quick PR :)

Reply:
see https://github.com/devtools-html/perf.html/pull/1113